packet/utils.py

```diff
@@ -21,6 +21,7 @@ def before_request(func):
     @wraps(func)
     def wrapped_function(*args, **kwargs):
         uid = str(session['userinfo'].get('preferred_username', ''))
+        member = ldap_get_member(uid)
```

Comment:
Will ldap get member return None for a bad uid, or does it throw errors? I thought it threw errors, but I'm not certain.